nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         activation: str = 'relu',
         log_softmax: bool = True,
         dropout: float = 0.0,
-        use_transformer_pretrained: bool = True,
+        use_transformer_init: bool = True,
```

Comment:
use Optional

Reply:
Optional is a misleading word. I think Optional means Union[X, None]. It is used just when default value is None.

Reply:
oh good to know